### ISSUE TITLE
Calling flow kickstart:site from a directory different to the base directory 

### ISSUE THREAD

**Issue by RobinJespersen** [neos/flow-development-collection#1778]
### Description
If I call ./flow kickstart:site from within the directory containing also the related composer.json  file, everything is fine.

But if I call ./test/flow kickstart:site (from one directory above) it fails with the following message:

```
Notice: Undefined index: package/name in /test/Packages/Framework/Neos.Flow/Classes/Package/PackageManager.php line 391

  Type: Neos\Flow\Error\Exception
  Code: 1
  File: Packages/Framework/Neos.Flow/Classes/Error/ErrorHandler.php
  Line: 81
```
And creates an empty composer.json file in the directory above the ./flow file. 

### Steps to Reproduce

1. composer create-project neos/neos-base-distribution test
2. ./test/flow kickstart:site Package.Name test

#### Expected behavior

` ./test/flow kickstart:site Package.Name test`
should be equivalent to 
` cd ./test && ./flow kickstart:site Package.Name test`

### Affected Versions
site-kickstarter: 4.3.4
Neos: 4.3.4
Flow:5.3.4

**Comment by davidspiola** [comment]
@RobinJespersen we can close this issue, right?

**Comment by eWert-Online** [comment]
I would say this can be closed 😊
It was fixed by #1837

**Comment by RobinJespersen** [comment]
thx :)
